### PR TITLE
Update the Bullet Train template

### PR DESCRIPTION
**Pull request by juliancheal**
Updated the Avo community gem version. Pulling gem from rubygems
Configured the root path to be consistent with setting up a new Bullet Train App
Changed team resource match the code in the current Bullet Train repo

Fixed UsersCount card https://github.com/avo-hq/avo_upgrade/issues/6

### CODE REVIEW

**Comment by juliancheal** [comment]
I realised in the `resources` for `team`, `membership`, `invitation` in the Bullet Train the `self.search` is comment out. Should I also remove those from the template?

File: template/app/avo/resources/team.rb [hunk 1]
```diff
@@ -12,13 +12,13 @@ class Avo::Resources::Team < Avo::BaseResource
...
-    field :time_zone, as: :text
-    field :locale, as: :text
-    field :users, as: :has_many, through: :memberships
+    field :email, as: :text
+    field :first_name, as: :text
+    field :last_name, as: :text
+    field :time_zone, as: :select, options: -> { view_context.time_zone_options_for_select(Avo::Current.user.time_zone, nil, ActiveSupport::TimeZone) }
+    field :current_team, as: :belongs_to
+
+    field :teams, as: :has_many, through: :teams
```
Comment (adrianthedev) [inline template/app/avo/resources/team.rb]: These fields don't seem right for teams.

File: template/app/avo/resources/team.rb [hunk 2]
```diff
@@ -12,13 +12,13 @@ class Avo::Resources::Team < Avo::BaseResource
...
     field :memberships, as: :has_many
```
Comment (adrianthedev) [inline template/app/avo/resources/team.rb]: Taken from https://github.com/bullet-train-co/bullet_train/blob/main/app/avo/resources/team.rb

```suggestion
    field :name, as: :text
    field :slug, as: :text
    field :being_destroyed, as: :boolean, hide_on: :forms
    field :time_zone, as: :select, options: -> { view_context.time_zone_options_for_select(Avo::Current.user.time_zone, nil, ActiveSupport::TimeZone) }
    field :locale, as: :text
    field :users, as: :has_many, through: :memberships
    field :memberships, as: :has_many
    field :invitations, as: :has_many
```

**Comment by adrianthedev** [comment]
Thanks for the contribution @juliancheal!

**Comment by adrianthedev** [comment]
We ship the search option commented out so the user knows it's there and they can enable it.

**Pull request merged**